types/react-html-email/react-html-email-tests.ts

```diff
@@ -0,0 +1,10 @@
+import { createElement } from 'react';
+import { Email, renderEmail } from 'react-html-email';
+
+// $ExpectType string
+const email = renderEmail(
+	createElement(
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-html-email/react-html-email-tests.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.